buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -200,6 +235,15 @@ class BuildPlugin implements Plugin<Project>  {
         project.sourceCompatibility = '1.8'
         project.targetCompatibility = '1.8'
 
+        // TODO: Remove all root project distribution logic. It should exist in a separate dist project.
```

Comment:
This is a problem that I'm working on addressing in another PR. The root project is all sorts of messed up because it depends (without using dependencies in a lot of cases 😱) on all the subprojects to build an uber jar. This uberjar/dist zip should be generated in a subproject instead of the root project, using dependencies, configurations, and all other appropriate build script hygiene to resolve the cross project links.